date.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@/bin
+libdir=@CMAKE_INSTALL_PREFIX@/lib@LIB_SUFFIX@
```

Comment:
It should be `CMAKE_INSTALL_LIBDIR` for `libdir`, and `CMAKE_INSTALL_INCLUDEDIR` for `includedir`. Also I am not sure for `exec_prefix`.

Jsoncpp provides a good example of a pkg-config file: https://github.com/open-source-parsers/jsoncpp/blob/master/pkg-config/jsoncpp.pc.in.

Reply:
Thanks, today I learned: https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html.

Fixed, tested e2e with `waybar`. Pushed.

Reply:
I don't really understand the nuances of cmake here, so I don't know if the prefixes are necessary (or are even used in `date`s build). But I at least corrected the weird construction I was doing.